presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -15,8 +15,8 @@ Synopsis
     [ HAVING condition]
     [ { UNION | INTERSECT | EXCEPT } [ ALL | DISTINCT ] select ]
     [ ORDER BY expression [ ASC | DESC ] [, ...] ]
-    [ LIMIT { count | ALL } ]
     [ OFFSET count [ ROW | ROWS ] ]
+    [ LIMIT { count | ALL } ]
     [ FETCH { FIRST | NEXT } [ count ] { ROW | ROWS } { ONLY | WITH TIES } ]
```

Comment:
They are mutually exclusive and so should be separated by `|`.

Reply:
I couldn't find a good way to include that here while also keeping it easy to read. This top section is intended to be a synopsis. This restriction is documented in the section describing the feature.